src/themes/money/theme.json

```diff
@@ -818,6 +822,29 @@
         "variant": "compounds.cookie-banner.button-with-checkbox",
         "borderColor": "plum"
       }
+    },
+
+    "heroCard": {
+      "variants": {
+        "background": "linear-gradient(161.42deg, #924A8B 22.95%, #AF4C83 77.05%)",
+        "box-shadow": "0px 9px 19px rgba(92, 36, 87, 0.2)",
+        "column-gap": 104,
+        "paddingX":["sm","xxxl"],
+        "pt": ["md", 85],
```

Comment:
looks like some magic numbers, are we not using standard theme spacing?

Reply:
@jonathan-fielding there's no number close enough to what design has given me